specification/search/resource-manager/Microsoft.Search/preview/2021-04-01-preview/search.json

```diff
@@ -0,0 +1,2441 @@
+              },
+              {
+                "value": "Unspecified",
+                "name": "Unspecified",
+                "description": "Enforcement policy is not explicitly specified, with the behavior being the same as if it were set to 'Disabled'."
+              }
+            ]
+          }
+        },
+        "encryptionComplianceStatus": {
+          "readOnly": true,
+          "description": "Describes whether the search service is compliant or not with respect to having non customer encrypted resources. If a service has more than one non customer encrypted resource and 'Enforcement' is 'enabled' then the service will be marked as 'nonCompliant'.",
+          "type": "string",
+          "enum": [
+            "Compliant",
+            "NonCompliant"
+          ],
+          "x-ms-enum": {
+            "name": "SearchEncryptionComplianceStatus",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "Compliant",
+                "name": "Compliant",
+                "description": "Indicates that the search service is compliant, either because number of non customer encrypted resources is zero or enforcement is disabled."
+              },
+              {
+                "value": "NonCompliant",
+                "name": "NonCompliant",
+                "description": "Indicates that the search service has more than 1 non customer encrypted resources."
+              }
+            ]
+          }
+        }
+      },
+      "description": "Describes a policy that determines how resources within the search service are to be encrypted with Customer Managed Keys."
+    },
+    "PrivateEndpointConnection": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateEndpointConnectionProperties",
+          "description": "Describes the properties of an existing Private Endpoint connection to the Azure Cognitive Search service."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "Describes an existing Private Endpoint connection to the Azure Cognitive Search service."
+    },
+    "NetworkRuleSet": {
+      "properties": {
+        "ipRules": {
+          "type": "array",
+          "description": "A list of IP restriction rules that defines the inbound network(s) with allowing access to the search service endpoint. At the meantime, all other public IP networks are blocked by the firewall. These restriction rules are applied only when the 'publicNetworkAccess' of the search service is 'enabled'; otherwise, traffic over public interface is not allowed even with any public IP rules, and private endpoint connections would be the exclusive access method.",
+          "items": {
+            "$ref": "#/definitions/IpRule"
+          }
+        },
+        "byPass": {
```

Comment:
>byPass [](http://example.com/codeflow?start=9&length=6)

Is it "by pass" or "bypass". If it's the latter, you should fix the casing.

Reply:
It is the latter, thanks for catching it, will fix here and below!